TemporalAirlinesConcept.Common/Extensions/TemporalClientExtensions.cs

```diff
@@ -0,0 +1,24 @@
+﻿using Temporalio.Client;
+
+namespace TemporalAirlinesConcept.Common.Extensions;
+
+public static class TemporalClientExtensions
+{
+    public static async Task<bool> IsWorkflowRunning<T>(this ITemporalClient client, string workflowId)
```

Comment:
No value in the type parameter here or in the handle extensions

Reply:
Added for ease of use and to reduce duplication. It calls another extension with a handle statuses check

Reply:
```suggestion
    public static async Task<bool> IsWorkflowRunning(this ITemporalClient client, string workflowId)
```

I am saying everywhere in these extensions you are accepting a `WorkflowHandle` with a type parameter unnecessarily, you can just accept `WorkflowHandle`.